pkg/api/api.proto

```diff
@@ -64,6 +66,31 @@ message DeleteEnvironmentApplicationLockRequest {
   string lock_id = 3;
 }
 
+
+message CreateNewReleaseRequest {
+  string environment = 1;
+  string application = 2;
+  optional string team = 3;
+  // env->yaml
+  map<string, string> manifests = 4;
+  optional uint64 version = 5;
+  string sourceCommitId = 6;
+  string sourceAuthor = 7;
+  string sourceMessage = 8;
+  optional string sourceRepoUrl=9;
+}
+
+message CreateNewReleaseResponse {
+  repeated string deployedToEnvironments = 1;
+  string application = 2;
```

Comment:
Is there a good reason we echo back the application name? I never seen anyone using this.

Reply:
For the app name, not really.
I would however, like to return *something*, e.g. where this release was deployed to immediately. This requires a bit of a refactoring, because the transformers don't return anything though.

Reply:
For now, this is handled in server.go:ServeHTTP:~400

Reply:
I removed it for now.